components/measure-upload/MeasureRepositoryUpload.tsx

```diff
@@ -120,10 +103,26 @@ export default function MeasureRepositoryUpload({ logError }: MeasureUploadProps
   */
   async function retrieveMeasures() {
     setIsLoadingIds(true);
+    let parsedMeasureRepositoryUrl = measureRepositoryUrl;
+    if (measureRepositoryUrl.slice(-1) === '/') {
```

Comment:
If I enter a measure repository url and then delete it, the "Get Measures" button is grayed out. However, for me the "Get Measures" button is not grayed out when I first load or refresh the app. If I click on "Get Measures" without a url, I get a TypeError because it's trying to slice a nonexistent url:

![Screenshot 2023-04-06 at 1 34 01 PM](https://user-images.githubusercontent.com/87094479/230453655-f5c6bbc8-8abf-4b0a-8be0-69af0e3245e2.png)

We should make the button grayed out on initial load if possible, or at least do a check here to see that the url is defined

Reply:
Wait actually I am having a hard time replicating this error